docs/user_interface/how_to_deploy_ui.rst

```diff
@@ -0,0 +1,109 @@
+================
+How To Deploy UI
+================
+
+The PARETO UI is deployed using Electron.js. Visit `their site <https://www.electronjs.org/docs/latest/>`_ for complete documentation.
+
+The deployment process is dependent on the operating system in use. We currently support :ref:`windows_guide_start` and :ref:`mac_guide_start`.
+
+**Prerequisites**
+
+- Miniconda
+- npm
+- Active conda environment with PARETO UI installed locally. To install the PARETO UI locally, follow the steps in our guide :ref:`how-to-install-ui-locally-page`
```

Comment:
Due to licensing issues that recently came up, I think you should actually add a link here to conda-forge https://conda-forge.org/ and call it that by name. People shouldn't be using Conda any more unless they know what they're doing.

Reply:
also that means I would call the section "Miniforge"

Reply:
@MichaelPesce I'm a little confused by this - what is Miniforge and why is it required instead of Minconda?

Reply:
@tarnold17 My understanding (which may not be entirely correct) is it is a version of miniconda that only uses the conda-forge channel. Apparently, miniconda included something in a recent update that indicated there may be some issues with using some of the other channels. In practice, it is no different for us because we can rely on the conda-forge channel to install our python packages